app/Http/Controllers/Api/V1/RoundController.php

```diff
@@ -33,6 +33,58 @@ public function show(Round $round): RoundResource
     {
         return new RoundResource($round);
     }
+
+    public function store(Request $request)
+    {
+        $data = $request->validate([
+            'when' => 'required|date', // Ensure it is a valid date
+            'spots' => 'required|integer',
+            'course_id' => 'required|exists:courses,id',
+        ]);
+    
+        // Convert 'when' to the correct format
+        $data['when'] = (new \DateTime($data['when']))->format('Y-m-d H:i:s');
+    
+        // Set the host_id to the authenticated user's ID
+        $data['host_id'] = Auth::id();
+    
+        // Increment spots by 1 to account for the host
+        $data['spots'] += 1;
```

Comment:
I believe 'spots' is supposed to refer to the target number of golfers the host wants for the round... not the number of spots left.  Whenever we need to know how many free spaces are left, we should calculate it... number of spots substracted by the number of golfers (users in our case) attached to the round.  Guess this is a long way of saying I don't believe we want to increment the number of spots here.